coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/metadata/sql/PostgresDatasetMap.scala

```diff
@@ -374,6 +374,25 @@ class PostgresDatasetMapReader[CT](val conn: Connection, tns: TypeNamespace[CT],
     }
   }
 
+  def datasetInfoByResourceNameQuery = "SELECT system_id, next_counter_value, locale_name, obfuscation_key, resource_name FROM dataset_map WHERE resource_name = ?"
```

Comment:
Yeah, when we added resource_name to truth... we weren't thinking about doing lookups with it. Probably makes sense to now materialize the column. Though, it will be a tricky migration, but it worked last time (when we shut down all the secondary-watcher etc)